Tools/MiniBrowser/playstation/MainWindow.cpp

```diff
@@ -106,7 +115,45 @@ MainWindow::MainWindow(const char* requestedURL)
     }});
 
     createNewWebView(nullptr);
-    activeWebView()->loadURL(requestedURL ? requestedURL : "https://webkit.org");
+
+    // Parse command line options
+    string requestedURL;
+
+    for (auto option : options) {
+        bool foundOption = false;
+        __debugbreak();
+        for (auto optionName : s_optionNames) {
+            int len = strlen(optionName);
+            if (!option.compare(0, len, optionName)) {
```

Comment:
You compare a string twice here and executeOption. It looks a bit nonsence.
if (option == "--resource-load-statistics-enabled") {
} else if (option == "--resource-load-statistics-disabled")